src/org/loklak/tools/JsonCompatHelper.java

```diff
@@ -1,51 +1,58 @@
 /**
- *  JsonCompatHelper
- *  Copyright 06.06.2016 by Marc Nause, @nause_marc
- *
- *  This library is free software; you can redistribute it and/or
- *  modify it under the terms of the GNU Lesser General Public
- *  License as published by the Free Software Foundation; either
- *  version 2.1 of the License, or (at your option) any later version.
- *
- *  This library is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; wo even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- *  Lesser General Public License for more details.
- *
- *  You should have received a copy of the GNU Lesser General Public License
- *  along with this program in the file lgpl21.txt
- *  If not, see <http://www.gnu.org/licenses/>.
+ * JsonCompatHelper
+ * Copyright 06.06.2016 by Marc Nause, @nause_marc
+ * <p>
```

Comment:
Sorry this happened by mistake. JsonCompatHelper class has 2 spaces as indentation, while fixing that IDE automatically changed it. I will remove the `<p>` tags from the header.